e2e/tests/cohorts/cohorts-create.spec.ts

```diff
@@ -46,14 +46,13 @@ describe('User can create new Cohorts', () => {
     const group1Count = await group1.includePhysicalMeasurement(PhysicalMeasurementsCriteria.BMI, 30);
 
     // Checking Group 1 Count. should match Group 1 participants count.
-    await waitForText(page, group1Count, {xpath: group1.getGroupCountXpath()});
-    const group1CountInt = Number(group1Count.replace(/,/g, ''));
+    const group1CountInt = Number((await group1.getGroupCount()).replace(/,/g, ''));
```

Comment:
Question: Why was waitForText removed?

Reply:
Because you added await?

Reply:
Replaced it with call to `await group1.getGroupCount()` to find Group Count.